vATIS Profile - Hong Kong FIR VHHK.json

```diff
@@ -360,8 +423,8 @@
         "closingStatement": {
           "autoIncludeClosingStatement": true,
           "template": {
-            "text": "ACKNOWLEDGE INFO {letter} ON FIRST CTC.",
-            "voice": "ACKNOWLEDGE INFORMATION {letter|word} ON FIRST CONTACT."
+            "text": "ACKNOWLEDGE INFO {letter}, ON FIRST CTC.",
+            "voice": "ACKNOWLEDGE INFORMATION {letter|word}, ON FIRST CONTACT."
```

Comment:
Does the comma exist in the text version too?

Reply:
That is an error, fixed